docs/testing.md

```diff
@@ -16,16 +16,22 @@ Included with forklift is a small live test suite.  The current tests are:
   * fb-destroy-organization.bats - Cleans up after the content tests
   * fb-finish.bats - Collects logs pertinent to the bats run
 
-To execute the bats framework:
+## Executing the Bats pipeline
 
- * Using vagrant (after configuring vagrant according to this document):
+ ### Using Vagrant
 
+ (after configuring vagrant according to this document):
+
+```
     vagrant up centos7-pipeline-bats
+```
 
-To run the same setup run by CI system:
+### To run the same setup run by CI system
 
+```
     cp boxes.yaml.example boxes.yaml
     vagrant up centos7-bats-ci
```

Comment:
If you add backticks, you should remove the indenting. While you're at it, a5ef2c82a0201f7667cf1ecd739ad8dee0720cf3 renamed the box
```suggestion
cp boxes.yaml.example boxes.yaml
vagrant up centos7-katello-bats-ci
```